src/search.c

```diff
@@ -610,93 +610,52 @@ static double query_total_download_size(struct list *list)
 }
 
 /* download_manifests()
- * Description: To search Clear bundles for a particular entry, a complete set of
- *		manifests must be downloaded. This function does so, asynchronously, using
- *		the curl_multi interface */
-static enum swupd_code download_manifests(struct manifest **MoM)
+ * Download all manifests and return a list of manifest headers.
+ */
+static enum swupd_code download_all_manifests(struct manifest *mom)
 {
-	struct list *list = NULL;
-	struct file *file = NULL;
-	char *tarfile, *untard_file, *url = NULL;
-	struct manifest *subMan = NULL;
-	int current_version;
+	struct manifest *m = NULL;
```

Comment:
I think we should avoid using such short variable names, it gets confusing when reading the code. For example in this case you could get confused if `m` holds the MoM or a manifest. I prefer more explicit variable names.